### PR TITLE
Some FAQ and motd update

### LINKED ISSUE #3087: Services containers don't report startup failure
Foreword: 
I love linuxkit!

But:
If for any reason a service container fails to start, there's no indication of the issue in the console nor in any log (that I could find). Is there're a way to make `containerd` log what it is doing during boot in the console or in a log file?

**Steps to reproduce the issue:**

For instance, the following linuxkit.yml
~~~yml
kernel:
  image: linuxkit/kernel:4.9.87
  cmdline: "console=tty0 console=ttyS0 console=ttyAMA0"
init:
  - linuxkit/init:6061875ba11fd9c563fda6234b103ed9997ff782
  - linuxkit/runc:52ecfdef1ae051e7fd5ac5f1d0b7dd859adff015
  - linuxkit/containerd:13f62c61f0465fb07766d88b317cabb960261cbb
onboot:
  - name: metadata
    image: linuxkit/metadata:ae402c118338c0285625c0094525eb0a90085a58
  - name: dhcpcd
    image: linuxkit/dhcpcd:v0.2
    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
  - name: sysctl
    image: linuxkit/sysctl:v0.4
  - name: sysfs
    image: linuxkit/sysfs:v0.4
  - name: binfmt
    image: linuxkit/binfmt:0f73915a3fe365615a10a811f5b83303f48949d5
# Format and mount the disk image in /var/lib for persistent mysql
  - name: format
    image: linuxkit/format:v0.4
  - name: mount
    image: linuxkit/mount:v0.4
    command: ["/usr/bin/mountie", "/var/lib"]
# make a swap file on the mounted disk
  - name: swap
    image: linuxkit/swap:v0.4
    command: ["/swap.sh", "--path", "/var/lib/swap", "--size", "1024M"]
services:
  # Enable acpi to shutdown on power events
  - name: acpid
    image: linuxkit/acpid:v0.4
  # Enable getty for easier debugging
  - name: getty
    image: linuxkit/getty:44730fd0a7c59dbacf5b48b54ba33f551bcf7ef0
    env:
        - INSECURE=true
  # Run ntpd to keep time synchronised in the VM
  - name: ntpd
    image: linuxkit/openntpd:v0.4
  - name: sshd
    image: linuxkit/sshd:v0.2
    env:
      - TINI_SUBREAPER=true
    capabilities:
     - all
    net: host
    pid: host
    binds:
     - /root/.ssh/authorized_keys:/root/.ssh/authorized_keys
     - /etc/resolv.conf:/etc/resolv.conf
  # Monitor for image deletes and invoke a TRIM on the container filesystem
  - name: trim-after-delete
    image: linuxkit/trim-after-delete:v0.4
  # When the host resumes from sleep, force a clock resync
  - name: host-timesync-daemon
    image: linuxkit/host-timesync-daemon:v0.4
  # The stuff we're running
  - name: mysql
    image: mysql:5.7
    command: ['mysqld', '--character-set-server=utf8', '--collation-server=utf8_unicode_ci' , '--innodb-flush-log-at-trx-commit=0', '--innodb-flush-method=nosync']
    capabilities:
      - all
    net: host
    env:
      - MYSQL_ROOT_PASSWORD=rootpw
      - MYSQL_DATABASE=acceptance_dow
      - MYSQL_USER=admin
      - MYSQL_PASSWORD=admin
    binds:
      - /etc/resolv.conf:/etc/resolv.conf
      - /var/lib/mysql:/var/lib/mysql
files:
  - path: /root/.ssh/authorized_keys
    source: "files/ssh/mykey.pub"
    mode: "0600"
  - path: /var/lib/mysql
    directory: true
    uid: 999
    gid: 999
    mode: "0750"
trust:
  org:
    - linuxkit
    - library
~~~

will provide an image that when run has no mysql process running.
(spoiler alert: the reason is that the oci runtime doesn't find `mysqld` in the path)

**Describe the results you received:**

Logging into the (qemu) based VM, no mysql is running, nothing in the output or error log:
~~~sh
linuxkit-befde23bc535:/# cat /var/log/mysql.out.log
linuxkit-befde23bc535:/# cat /var/log/mysql.err.log
linuxkit-befde23bc535:/# ctr -n services.linuxkit c ls
CONTAINER               IMAGE    RUNTIME
acpid                   -        io.containerd.runtime.v1.linux
getty                   -        io.containerd.runtime.v1.linux
host-timesync-daemon    -        io.containerd.runtime.v1.linux
mysql                   -        io.containerd.runtime.v1.linux
ntpd                    -        io.containerd.runtime.v1.linux
sshd                    -        io.containerd.runtime.v1.linux
trim-after-delete       -        io.containerd.runtime.v1.linux
linuxkit-befde23bc535:/# ctr -n services.linuxkit t ls
TASK                    PID    STATUS
sshd                    845    RUNNING
acpid                   583    RUNNING
getty                   629    RUNNING
host-timesync-daemon    675    STOPPED
ntpd                    773    RUNNING
~~~

So the container refused to start, and there's no indication in the boot console, nor in any log why it failed.

We have to try to start it manually to view the error:
~~~sh
linuxkit-befde23bc535:/# ctr -n services.linuxkit t start mysql
ctr: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"mysqld\": executable file not found in $PATH": unknown
~~~

**Describe the results you expected:**

During boot, I would have expected to see any `containerd` error or failure in a log or in the console to help troubleshoot container issues.

**Additional information you deem important (e.g. issue happens only occasionally):**

Also, it would be great to have documentation on how to troubleshoot `containerd` in the docs, as it took me a while to find about the `ctr` command and how it works.

### PR DESCRIPTION
**Pull request by masterzen**
fixes #3087 

**- What I did**

I fixed the `getty` and `sshd` `motd` to report the correct `ctr` command usage, as the indicated namespace is incorrect.
I also added two new FAQ entries adding a bit more information that I gathered while troubleshooting my issue in #3087. I think this could help other users as not everybody knows exactly how `linuxkit` or `containerd` works.

**- How I did it**

With my hands, my brain and a text editor :)

**- How to verify it**

I'm not a native English speaker, so you probably should proof-read what I wrote :)

**- Description for the changelog**

Fixed incorrect namespace in motd
Enhanced FAQ with troubleshooting advices

**- A picture of a cute animal (not mandatory but encouraged)**

wombat

Thanks !

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -30,3 +30,51 @@ of dependencies and functionality that we do not need. At present we are using t
...
+
+## Console not displaying init or containerd output at boot
+
+If you're not seeing `containerd` logs in the console during boot, make sure that your kernel `cmdline` configuration doesn't list multiple consoles.
+
+`init` and other processes like `containerd` will use the last defined console in the kernel `cmdline`. When using `qemu`, to see the console you need to list `ttyS0` as the last console to properly see the output.
+
+## Troubleshooting containers
+
+Linuxkit runs all services in a specific namespace `services.linuxkit`. When entering the console, the logged in user is in the `getty` namespace.
```
Comment (rn) [inline docs/faq.md]: The word "namespace" is a bit overloaded and confusing here There are `containerd` namespaces (like `services.linuxkit`) and there are Linux namespace (`getty` runs in one). Maybe rephrase:

"LinuxKit runs all services in a specific `containerd` namespace called `services.linuxkit`. To list all defined containers use:"

Comment (masterzen) [reply docs/faq.md]: @rn, thanks, that's indeed better.
I just pushed a fix.

**Comment by masterzen** [comment]
@rn thanks for merging this PR!
